WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -91,8 +95,10 @@ enum DashboardCard: String, CaseIterable {
             return BlazeHelper.shouldShowCard(for: blog)
         case .domainsDashboardCard:
             return DomainsDashboardCardHelper.shouldShowCard(for: blog)
+        case .empty:
+            return false // Controlled manually based on other cards visibility
         case .personalize:
-            return AppConfiguration.isJetpack && FeatureFlag.personalizeHomeTab.enabled
```

Comment:
I removed `AppConfiguration.isJetpack` from here to make it easier to test using the WordPress test target. I'm planning to add `AppConfiguration.isJetpack` check to `FeatureFlags` instead one we are ready to enable it, the same way it's done for prompts, for example:

```swift
case .bloggingPrompts:
    return AppConfiguration.isJetpack
```